collector/README.md

```diff
@@ -11,20 +11,8 @@ The primary components are:
 
 ## Building and distributing these components
 
-We are aware that building and distributing OpenTelemetry collectors
-is not a simple task and have prepared dedicated instructions for
-building and testing the components in this repository.
-
-[Instructions for building an OpenTelemetry Collector with support for
-OpenTelemetry Protocol with Apache Arrow.][BUILDING]
-
-After you have built a collector using one of the documented methods,
-see the [examples][EXAMPLES].
-
-[We would prefer to include these components in the OpenTelemetry
-Contrib Collector, because it is an officially maintained artifact.
-At this time, however, these components are new and the migration
-process will take some time to complete.][CONTRIBUTION]
+The exporter and receiver components are included in the official
+OpenTelemetry Collector-Contrib release images since v0.105.0.
```

Comment:
I suggest adding a link to the configuration/installation procedure or to the OpenTelemetry contribs collector builder.

Reply:
I also noticed some broken links remaining here, will revise.

Reply:
I will do this in a new PR, since this one is fragile.